netlogo-gui/src/main/ide/CodeCompletionPopup.scala

```diff
@@ -38,6 +38,20 @@ case class CodeCompletionPopup() {
       suggestionDisplaylist.setLayoutOrientation(JList.VERTICAL)
       suggestionDisplaylist.setVisibleRowCount(10)
       suggestionDisplaylist.setCellRenderer(new SuggestionListRenderer(eA))
+      
+      suggestionDisplaylist.addMouseListener(new MouseListener {
+        override def mouseExited(e: MouseEvent): Unit = {}
+
+        override def mouseClicked(e: MouseEvent): Unit = {}
+
+        override def mouseEntered(e: MouseEvent): Unit = {}
+
+        override def mousePressed(e: MouseEvent): Unit = {}
+
+        override def mouseReleased(e: MouseEvent): Unit = {
```

Comment:
@qiemem I usually write code in `mouseReleased` because in my experience the place where I release the mouse is usually the right one. Let me know if this has to be changed.